src/terraform/commands.go

```diff
@@ -33,6 +33,21 @@ func RunPlan(deployConfig types.DeployConfig, secrets types.Secrets) error {
 	return err
 }
 
+// RunApply runs the 'terraform apply' command and passes variables in as command flags
+func RunApply(deployConfig types.DeployConfig, secrets types.Secrets) error {
+	command := []string{"terraform", "apply"}
+	vars, err := CompileVarFlags(deployConfig, secrets)
+	if err != nil {
+		return err
+	}
+	command = append(command, vars...)
+	err = util.RunAndLog(deployConfig.TerraformDir, []string{}, deployConfig.LogChannel, command...)
+	if err != nil {
+		return errors.Wrap(err, "'terraform apply' failed")
```

Comment:
I don't think this wrap is needed as its wrapping it with arguments passed into `util.RunAndLog`. Ideally we only wrap it with context not passed to the function and rely on the function to report good errors based on its inputs. 

`util.RunAndLog` already wraps run errors with the command it was running and the output